internal/services/compute/linux_virtual_machine_resource_other_test.go

```diff
@@ -572,6 +572,50 @@ func TestAccLinuxVirtualMachine_otherGracefulShutdownEnabled(t *testing.T) {
 	})
 }
 
+func TestAccLinuxVirtualMachine_otherSecureBootEnabled(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_linux_virtual_machine", "test")
+	r := LinuxVirtualMachineResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.otherSecureBootEnabled(data, false),
```

Comment:
which means this should be Enabled true:

```suggestion
			Config: r.otherSecureBootEnabled(data, true),
```